### PR TITLE
[risk=low][no ticket] Remove some unused ruby

### PR DESCRIPTION
**Pull request by jmthibault79**
We don't seem to need any of these.  Removing to avoid confusion.

---
**PR checklist**

- [x] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [x] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have manually tested this change and my testing process is described above.
- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [ ] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [x] None of the above apply to this change.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -343,12 +343,6 @@ configuration uses the port the API is listening on, *8001*.
 When this configuration is selected, clicking the debug icon (looks like a green bug) will
 cause the running API process to stop at breakpoints you have set for inspection.
 
-## DevOps
-The `devops` toolkit is described [here](ops/ruby/devops-framework/README.md). Other
```
Comment (jmthibault79) [inline README.md:347]: not actually used.  No references in the Oncall Playbook or any scripts

File: ops/ruby/devops-toolkit/Gemfile [hunk 2]
```diff
@@ -1,4 +0,0 @@
-source 'https://rubygems.org'
```
Comment (jmthibault79) [inline ops/ruby/devops-toolkit/Gemfile:1]: the whole `ops/ruby` directory is unused

File: aou-utils/utils/common.rb [hunk 3]
```diff
@@ -87,14 +85,6 @@ def handle_or_die(args)
     end
   end
 
-  def load_env()
```
[the remark this answers is not in the document]
Comment (NehaBroad) [reply aou-utils/utils/common.rb:90]: It was introduced in this (i think)
https://github.com/all-of-us/workbench/commit/19822574e52ef4cbacefb087deb112f615d8a803

[the remark this answers is not in the document]
Comment (jmthibault79) [reply aou-utils/utils/common.rb:90]: Thanks.  Looks like it was just SyncFiles

**Review by NehaBroad — APPROVED** [review]

**Pull request merged**